src/main/java/org/ibp/api/rest/labelprinting/LotLabelPrinting.java

```diff
@@ -187,6 +188,11 @@ void validateLabelsGeneratorInputData(final LabelsGeneratorInput labelsGenerator
 		super.validateLabelsGeneratorInputData(labelsGeneratorInput, programUUID);
 	}
 
+	@Override
+	public LabelPrintingPresetDTO getDefaultSetting(final LabelsInfoInput labelsInfoInput, final String programUUID) {
+		return null;
+	}
```

Comment:
We can return Optional but it'll be more or less the same, as we need to return empty to the frontend